staging/src/k8s.io/apiserver/pkg/server/servermetrics.go

```diff
@@ -0,0 +1,48 @@
+/*
+Copyright 2020 The Kubernetes Authors.
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+package server
+
+import (
+	"net/http"
+
+	"github.com/prometheus/client_golang/prometheus"
```

Comment:
We don't actually use prometheus directly. This would have to be registered to our wrapper registry (https://github.com/kubernetes/kubernetes/blob/eda07adf6e3ef8d066ba95a9b1039a13d2e3f8ea/staging/src/k8s.io/apiserver/pkg/endpoints/metrics/metrics.go#L266, for instance).

Reply:
Should this documentation be changed? https://github.com/kubernetes/community/blob/master/contributors/devel/sig-instrumentation/instrumentation.md

Reply:
Yes, it should actually. Would you mind filing an issue so that we don't lose track of it?

Reply:
Done https://github.com/kubernetes/community/issues/5080